e2e/e2eutil/allocs.go

```diff
@@ -144,3 +144,25 @@ func AllocStatusesRescheduled(jobID string) ([]string, error) {
 	}
 	return statuses, nil
 }
+
+// AllocExec is a convenience wrapper that runs 'nomad alloc exec' with the
+// passed cmd, retrying if the task isn't ready
+func AllocExec(allocID, taskID string, cmd string, wc *WaitConfig) (string, error) {
+
+	//func AllocExec(params string, wc *WaitConfig) (string, error) {
+	var got string
+	var err error
+	interval, retries := wc.OrDefault()
+
+	args := append([]string{"alloc", "exec", "-task", taskID, allocID},
+		strings.Split(cmd, " ")...)
```

Comment:
I'd consider using `"/bin/sh", "-c", cmd` here and avoid parsing the command. If the command contains redirects, or quoted strings, or some sh-ism, the result will be somewhat unexpected.

Reply:
Good call.